### DISCUSSION TITLE
Can't load external scripts

### DISCUSSION THREAD

**Discussion by phillipstemann** [nuxt/nuxt#18267] [Questions] [answered]
Hi, I really hope someone can help me.
I'm playing around with nuxt3. I'm trying to load in feather icons in my nuxt config, but it doesn't load in the file when I run "yarn dev". Can anyone see what I'm doing wrong in my setup?

```import { defineNuxtConfig } from 'nuxt3'

export default defineNuxtConfig({
    script: [
      { src: "https://unpkg.com/feather-icons@4.28.0/dist/feather.min.js", body: true }
    ],
    css: ['~/assets/scss/main.scss']
})

**Comment by wobsoriano** [comment, marked as answer]
Use [meta](https://v3.nuxtjs.org/docs/directory-structure/nuxt.config#meta) instead

```ts
export default defineNuxtConfig({
  meta: {
    script: [
      { src: 'https://unpkg.com/feather-icons@4.28.0/dist/feather.min.js', defer: true }
    ],
  }
});
```

**Reply by phillipstemann** [reply]
You are awesome, thank you so much. Have a fantastic weekend.
